src/assets/scss/_tabs.scss

```diff
@@ -298,181 +337,161 @@
 }
 
 .nav-tabs-fillup {
-  -webkit-backface-visibility: hidden;
-  -moz-backface-visibility: hidden;
   backface-visibility: hidden;
-  -webkit-perspective: 1000;
-  -moz-perspective: 1000;
   perspective: 1000;
+
   > li {
     overflow: hidden;
+
     > a {
-      -webkit-backface-visibility: hidden;
-      -moz-backface-visibility: hidden;
       backface-visibility: hidden;
-      -webkit-transition: color 0.3s ease 0s;
-      transition: color 0.3s ease 0s;
       background: transparent;
-      &:after {
-        -webkit-backface-visibility: hidden;
-        -moz-backface-visibility: hidden;
+      transition: color 0.3s ease 0s;
+
+      &::after {
+        position: absolute;
+        z-index: -1;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: calc(100% + 1px);
+        border: 1px solid #10cfbd;
         backface-visibility: hidden;
         background: none repeat scroll 0 0 #10cfbd;
-        border: 1px solid #10cfbd;
         content: "";
-        height: calc(100% + 1px);
-        left: 0;
-        position: absolute;
-        top: 0;
-        -webkit-transform: translate3d(0, 100%, 0px);
-        transform: translate3d(0, 100%, 0px);
-        -webkit-transition: -webkit-transform 0.3s ease 0s;
+        transform: translate3d(0, 100%, 0);
         transition: transform 0.3s ease 0s;
-        width: 100%;
-        z-index: -1;
       }
-    }
-  }
-}
-
-.nav-tabs-primary.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #6d5cae;
-  border: 1px solid #6d5cae;
-}
-
-.nav-tabs-success.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #10cfbd;
-  border: 1px solid #10cfbd;
-}
-
-.nav-tabs-complete.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #48b0f7;
-  border: 1px solid #48b0f7;
-}
-
-.nav-tabs-warning.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #f8d053;
-  border: 1px solid #f8d053;
-}
-
-.nav-tabs-danger.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #f55753;
-  border: 1px solid #f55753;
-}
-
-.nav-tabs-info.nav-tabs-fillup > li > a:after {
-  background: none repeat scroll 0 0 #3b4752;
-  border: 1px solid #3b4752;
-}
 
-.nav-tabs-fillup {
-  > li {
-    > a span {
-      -webkit-backface-visibility: hidden;
-      -moz-backface-visibility: hidden;
-      backface-visibility: hidden;
-      -webkit-transform: translate3d(0px, 5px, 0px);
-      transform: translate3d(0px, 5px, 0px);
-      -webkit-transition: -webkit-transform 0.5s ease 0s;
-      transition: transform 0.5s ease 0s;
-      display: block;
-      font-weight: 700;
-      line-height: 1.5;
+      span {
+        display: block;
+        backface-visibility: hidden;
+        font-weight: 700;
+        line-height: 1.5;
+        transform: translate3d(0, 5px, 0);
```

Comment:
Looks like another instance where `translate3d` could be swapped for just `translate`.